.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
```

Comment:
:+1: Ah, that makes sense. I use gcc and libstdc++ on CI just because I normally use clang when I develop locally, and I want to be sure the library works with both toolchains.